rai/node/rpc.cpp

```diff
@@ -3526,6 +3580,94 @@ void rai::rpc_handler::wallet_key_valid ()
 	}
 }
 
+void rai::rpc_handler::wallet_ledger ()
+{
+	bool representative (false);
+	boost::optional<bool> representative_optional (request.get_optional<bool> ("representative"));
+	if (representative_optional.is_initialized ())
+	{
+		representative = representative_optional.get ();
+	}
+	bool weight (false);
+	boost::optional<bool> weight_optional (request.get_optional<bool> ("weight"));
```

Comment:
This behavior (`false` if maybe/undefined, `false` if defined `false`, `true` if defined `true` ) can be simplified as 

`const bool weight = *(request.get_optional<bool> ("weight"));`

Reply:
Ok, will change all similar parts as well